src/version.c

```diff
@@ -1748,6 +1748,11 @@ list_version(void)
     version_msg(USR_VIMRC_FILE3);
     version_msg("\"\n");
 #endif
+#ifdef XDG_VIMRC_FILE
+    version_msg(_(" 4th user vimrc file: \""));
+    version_msg(XDG_VIMRC_FILE);
```

Comment:
You need a `(char *)` cast here.

Reply:
@yegappan Thanks!

Reply:
Still failing, not sure.

Reply:
Looks like it's passing now, had to compile with `CC="gcc -Werror -Wpointer-sign"` to be able to reproduce the errors locally.